include/numerics.hpp

```diff
@@ -314,4 +313,82 @@ class Integrator1DFourier {
   
 };
 
+// --- Integrator for 1D integrals with known singularities --- 
```

Comment:
I am getting a compilation warning from line 328, please fix it

Reply:
Could I get a photo of these warnings? I just deleted build directory and re-built the code and I am not getting this warning.

Reply:
This is also something I will fix myself, closing also this issue